public/app/features/inspector/InspectStatsTable.tsx

```diff
@@ -44,22 +43,22 @@ export const InspectStatsTable: React.FC<InspectStatsTableProps> = ({ timeZone,
   );
 };
 
-function formatStat(stat: QueryResultMetaStat, timeZone?: TimeZone): string {
+function formatStat(stat: QueryResultMetaStat, timeZone: TimeZone, theme: GrafanaThemeV2): string {
   const display = getDisplayProcessor({
     field: {
       type: FieldType.number,
       config: stat,
     },
-    theme: config.theme,
+    theme,
     timeZone,
   });
   return formattedValueToString(display(stat.value));
 }
 
-const getStyles = stylesFactory((theme: GrafanaTheme) => {
+const getStyles = stylesFactory((theme: GrafanaThemeV2) => {
```

Comment:
Maybe remove usage of `stylesFactory` and use `useStyles2` instead?

Reply:
the theme is used elsewhere in the main component function, so would still have to use useTheme2